src/utils/adapters.ts

```diff
@@ -7,3 +7,11 @@ export const assetNetworkAdapter: Record<string, BlockchainEnum> = {
   arbitrum: BlockchainEnum.ARBITRUM,
   bitcoin: BlockchainEnum.BITCOIN,
 }
+
+export const reverseAssetNetworkAdapter: Record<BlockchainEnum, string> = {
```

Comment:
it's not just string, let's use `SupportedChainName`  from `src/types/base.ts:14`